x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/collapsed_item_actions.scss

```diff
@@ -0,0 +1,8 @@
+.actCollapsedItemActions__delete {
+  color: $euiColorDanger;
+
+  .euiContextMenu__itemLayout {
+    @include euiFontSizeS;
+    align-items: baseline;
+  }
```

Comment:
My guess is the overwrite for this one isn't necessary since you're able to pass your content inside (which can have its own selector)

Reply:
Using `EuiText` now